tests/kafkatest/tests/streams/streams_smoke_test.py

```diff
@@ -27,7 +27,7 @@ class StreamsSmokeTest(KafkaTest):
     """
 
     def __init__(self, test_context):
-        super(StreamsSmokeTest, self).__init__(test_context, num_zk=1, num_brokers=2, topics={
+        super(StreamsSmokeTest, self).__init__(test_context, num_zk=1, num_brokers=3, topics={
```

Comment:
It's actually needed for this PR since this PR changes the replication factor to 3 in SmokeTestClient.java. It just so happened that in trunk we had already done that change, so it appears as if it is part of KAFKA_4716, but it's part of this PR.